### ISSUE TITLE
Question about fulfillment.status

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9618]
What is the difference between READY_FOR_SHIPMENT and READY_FOR_PICKUP, SENT and PICKED_UP?

**Comment by Lukasz-Zurek** [comment]
I think these statuses speak for themselves:

READY_FOR_SHIPMENT - order to sent,
SENT - order sent,
READY_FOR_PICKUP - ready for pickup (e.g. in the pickup point)
PICKED_UP - shipment already picked up by buyer

**Comment by yangcen6** [comment]
So READY_FOR_PICKUP is for customer,not courier?

**Comment by Lukasz-Zurek** [comment]
@yangcen6 It is for the seller. You can check the details [here](https://developer.allegro.pl/tutorials/jak-obslugiwac-zamowienia-GRaj0qyvwtR#zmiana-statusu-realizacji-zamowienia).

**Comment by yangcen6** [comment]
![流程图 (16)](https://github.com/user-attachments/assets/2b0588e2-fe7d-4dbe-bbc8-acbe783b5e59)
This is the flow chart of order fulfillment status that I drew. Please help me check if it is correct.

**Comment by Lukasz-Zurek** [comment]
@yangcen6 Yes, you can use this way, but it us fully up to you. We have introduced all of these statuses to help with orders management.

**Comment by yangcen6** [comment]
Is it true that customers can only apply to cancel an order when the order status is **new**?

**Comment by Lukasz-Zurek** [comment]
Yes, we have described it [here](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#).

**Comment by yangcen6** [comment]
OK， thanks

**Comment by yangcen6** [comment]
Do we have to change the status to **’processing‘** before changing it to **’ready for shipment ‘**， or can we skip the process?

**Comment by MaciejFrackowiak** [comment]
You can skip it.

**Comment by yangcen6** [comment]
If we use ship with Allegro, after the package is delivered, will the order status on the Allegro automatically change to "delivered"?

**Comment by MaciejFrackowiak** [comment]
No. The order fulfillment.status will not change automatically.

**Comment by yangcen6** [comment]
But how should we inform customers that their packages have been delivered? If we use ship with Allegro, we have no connection with the carrier.

**Comment by MaciejFrackowiak** [comment]
The customer will receive an automatic notification that his/her package is delivered/waiting for collect.

**Comment by yangcen6** [comment]
Is this what Allegro send to customer after the package is delivered?

**Comment by yangcen6** [comment]
1. Do we have to change the status to ’ready for shipment‘ before changing it to ’sent ‘， or can we skip the process?
2. In other words, we have set up a subscription（https://allegro.pl/moje-allegro/sprzedaz/zamowienia/ustawienia?sellerId=123550430）,After we change the status to **processing**, do we no longer need to change the order status through [the interface](https://developer.allegro.pl/documentation/#operation/setOrderFulfillmentUsingPUT)? Because when a shipment is created though https://developer.allegro.pl/documentation/#operation/createNewShipment, the order will be added with a tracking number, and the fulfillment status will **automatically become sent**.
![20240820-202635](https://github.com/user-attachments/assets/1e70294e-650f-487a-96c4-b399774bbef0)

**Comment by MaciejFrackowiak** [comment]
1. It's up to you.
2. Yes, if you choose the selected option, fulfillment.status will be changed automatically after creating a shipment.

**Comment by yangcen6** [comment]
What time do we need to call /shipment-management/pickups/create-commands to call the courier to pickup our goods in warehouse? after we get the pickup proposals , or?

**Comment by MaciejFrackowiak** [comment]
Yes, it should be after the pickup date proposal request - we described the process in [our tutorial](https://developer.allegro.pl/tutorials/how-to-manage-parcels-via-ship-with-allegro-ZM9YAyGKWTV#how-to-request-shipment-pickup-by-a-courier).

**Comment by yangcen6** [comment]
1. When buyer has paid for the order, the **status** will become **READY_FOR_PROCESSING**. How about the **fulfillment.status** at this time? Automatically changed to **NEW**?
2. When we changed the fulfillment.status to PROCESSING, what status will be? still READY_FOR_PROCESSING?

**Comment by MaciejFrackowiak** [comment]
1. Yes, it will be NEW.
2. Yes, order status will be READY_FOR_PROCESSING.
